library/lib/components/dialogs/ods_alert_dialog.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter/components/button/ods_filled_button.dart';
+import 'package:ods_flutter/guidelines/spacings.dart';
+
+class DismissButtonDescription {
+  final String? dismissButtonText;
+  final Function()? onDismissButtonClick;
+
+  DismissButtonDescription({
+    this.dismissButtonText,
+    this.onDismissButtonClick,
+  });
+}
+
+class OdsAlertDialog extends StatefulWidget {
+  const OdsAlertDialog({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<OdsAlertDialog> createState() => _OdsAlertDialogState();
+
+  static void openDialog({
+    required BuildContext context,
+    required String text,
+    required String confirmButtonText,
```

Comment:
Just a question: perhaps we should do the same for confirmButtonText and onConfirmButtonClick, use the ButtonDescription ?